packages/dart/npt_flutter/lib/util/general_extensions.dart

```diff
@@ -0,0 +1,12 @@
+extension StringExtension on String {
+  String atsignify() {
+    var value = this;
+    if (!startsWith('@')) {
+      value = '@$this';
+    }
+    if (endsWith(' ')) {
+      value = trim();
```

Comment:
You should always trim and do it before checking startsWith, in case there is leading whitespace.